webgpu.yml

```diff
@@ -840,6 +840,8 @@ enums:
         doc: The system ran out of memory.
       - name: device_lost
         doc: The @ref WGPUDevice configured on the @ref WGPUSurface was lost.
+      - name: error
+        doc: The surface is not configured, or there was a @ref StructChainingError.
```

Comment:
Maybe just point at the Surface.md article for a description of what might return an error?

Reply:
I kind of think struct chaining isn't really relevant to pile into the Surface.md article and prefer to just document it here.